src/authentication.ts

```diff
@@ -0,0 +1,15 @@
+import { Secret, sign } from 'jsonwebtoken';
```

Comment:
`authentication.ts`->`auth.ts`

Reply:
It can stay as it is, but it's just unnecessarily long and `auth.file` is pretty standard.